app/script/view_model/list/StartUIViewModel.js

```diff
@@ -135,11 +135,10 @@ z.ViewModel.list.StartUIViewModel = class StartUIViewModel {
     this.hasCreatedConversation = ko.observable(false);
 
     // View states
-    this.hasSearchResults = ko.pureComputed(() => {
-      return (
+    this.hasSearchResults = ko.pureComputed(
+      () =>
         this.searchResults.groups().length || this.searchResults.contacts().length || this.searchResults.others().length
-      );
-    });
+    );
```

Comment:
I am definitely not a fan of these. If we need multiple lines anyway we should go for explicit returns. Now I have to check if this actually returns somthing...